### DISCUSSION TITLE
Publication using MFEM

### DISCUSSION THREAD

**Discussion by nickwhitman1993** [mfem/mfem#2116] [Show and tell]
Hi all, I just wanted to make it aware that I recently published an article using MFEM to produce results in case you wanted to add it to the publication list on the webpage. Thank you again for those that have been extremely helpful in the process of getting the correct bilinear and linear forms needed.

https://www.tandfonline.com/eprint/IFGD63KEIR9JKYHVHUXK/full?target=10.1080/23324309.2021.1900258

Best, 

Nick Whitman

**Comment by tzanio** [comment]
Thanks for sharing this Nick!

~You are welcome to open a PR for this in https://github.com/mfem/web (just edit [src/publications.md](https://github.com/mfem/web/blob/master/src/publications.md))~

It is live now: https://mfem.org/publications/

Tzanio

**Reply by nickwhitman1993** [reply]
Great, thanks @tzanio! 

Best,

Nick
